ci/test.sh

```diff
@@ -101,7 +101,7 @@ get_cargo_args() {
     args+=" --release"
   fi
   if [ "$BINDGEN_NO_DEFAULT_FEATURES" == "1" ]; then
-    args+=" --no-default-features"
+    args+=" --no-default-features --features prettyplease"
```

Comment:
I think it would be better if you leave this without `prettyplease` because the `test` workflow will check that `bindgen` compiles with `prettyplease` and `minimal` will test that `bindgen` compiles without `prettyplease`. This is just because you set `prettyplease` as a default feature, which is OK.

Reply:
removed. this PR still fails tests.

Reply:
can you rebase against current `main`... I expect that to result in passing tests

Reply:
done